vars/lookForGitHubIssues.groovy

```diff
@@ -16,46 +16,76 @@
 // under the License.
 
 /**
-  Look for all the open issues that were reported as flaky tests. It returns
+  Look for all the open issues given some filters.
+  
+  For backward compatibilities the default behaviour uses the flaky tests. It returns
   a dictionary with the test-name as primary key and the github issue if any or empty otherwise.
 
-  // Look for all the GitHub issues with label 'flaky-test' and test failures either test-foo or test-bar
-  lookForGitHubIssues( flakyList: [ 'test-foo', 'test-bar'], labelsFilter: [ 'flaky-test'])
+  // Look for all the open GitHub issues with label 'flaky-test' and test failures either test-foo or test-bar
+  lookForGitHubIssues(flakyList: [ 'test-foo', 'test-bar'], labelsFilter: [ 'flaky-test'])
+
+  // Look for all the open GitHub issues with label 'automation' and the title contains 'bump: stack'
+  lookForGitHubIssues(flakySearch: false, labelsFilter: ['automation'], titleContains: 'bump: stack')
 */
 def call(Map args = [:]) {
+  def flakySearch = args.get('flakySearch', true)
```

Comment:
This is the flag I added to change the default behaviour, in other words, I didn't want to change the existing features without affecting the consumers.